pkgs/applications/networking/remote/waypipe/default.nix

```
@@ -5,14 +5,14 @@
 
 stdenv.mkDerivation rec {
   pname = "waypipe-unstable";
```

```suggestion
  pname = "waypipe";
```
Does not look like it is unstable.

---

Right, it's a 0.x release but upstream now considers it "relatively stable":
> This program is now relatively stable, with no large changes to the
command line interface or wire format expected


Sources:
- https://gitlab.freedesktop.org/mstoeckl/waypipe/-/blob/ffd2d4379e481372ca6240f4b3bcedbd943dec4c/README.md#status
- https://gitlab.freedesktop.org/mstoeckl/waypipe/-/commit/ca1ca9440619e4a97012792a0a202b763d005f11